{{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings.py

```diff
@@ -72,6 +73,8 @@
 
 # Database
 # https://docs.djangoproject.com/en/1.9/ref/settings/#databases
+if not config('DATABASE_URL').startswith('sqlite') and not wait_for_url(config('DATABASE_URL')):
```

Comment:
I didn't think of doing it in settings... could work. I mostly conceived of using it as part of a script like `run-dev.sh`. You'd just put a call to `urlwait` before the command to run the site and it'd do the right thing.